modules.d/95nvmf/module-setup.sh

```diff
@@ -25,14 +25,24 @@ check() {
         [[ $trtype == "fc" ]] || [[ $trtype == "tcp" ]] || [[ $trtype == "rdma" ]]
     }
 
+    has_nbft() {
+        local f found=
+        for f in /sys/firmware/acpi/tables/NBFT*; do
+            [ -f "$f" ] || continue
+            found=1
+            break
+        done
+        [[ $found ]]
+    }
+
     [[ $hostonly ]] || [[ $mount_needs ]] && {
         pushd . > /dev/null
         for_each_host_dev_and_slaves is_nvmf
         local _is_nvmf=$?
         popd > /dev/null || exit
         [[ $_is_nvmf == 0 ]] || return 255
         if [ ! -f /sys/class/fc/fc_udev_device/nvme_discovery ]; then
-            if [ ! -f /etc/nvme/discovery.conf ]; then
+            if [ ! -f /etc/nvme/discovery.conf ] && ! has_nbft; then
```

Comment:
Shouldn't we also check here if `/etc/nvme/config.json` does not exist?

Reply:
Ooops. Thanks a lot.

Reply:
fixed with 9fd4901

Reply:
sorry, our "friend" `shfmt` is complaining: https://github.com/dracutdevs/dracut/actions/runs/5265185676/jobs/9517488413?pr=2184

Reply:
hm, I could swear I ran this locally. Anyway, hopefully fixed with 0a4d7f9